src/tests/ftest/unittest/Unittest.py

```diff
@@ -25,65 +25,79 @@
 
 from avocado.utils import process
 from general_utils import get_file_path
-from apricot import Test
+from apricot import Test, TestWithServers
 
-class UnitTest(Test):
+def unittest_runner(self, unit_testname):
     """
-    Avocado Unit Test class.
+    Common unitetest runner function.
+    Args:
+        unit_testname: unittest name.
+    return:
+        None
+    """
+    name = self.params.get("testname", '/run/UnitTest/{0}/'
+                           .format(unit_testname))
+    bin_path = get_file_path(name, "install/bin")
+
+    cmd = ("{0}".format(bin_path[0]))
+    return_code = process.system(cmd)
```

Comment:
Have you considered passing in `ignore_status=True` to the `process.system()` method?  It will raise a [`CmdError`](https://github.com/avocado-framework/avocado/blob/master/avocado/utils/process.py) exception that includes the command, stderr, and stdout.

Reply:
Done